src/lib/builders/tags-input/create.ts

```diff
@@ -325,6 +325,7 @@ export function createTagsInput(props?: CreateTagsInputProps) {
 				}),
 				addMeltEventListener(node, 'paste', async (e) => {
 					// Do nothing when there is nothing to paste
+					e.preventDefault();
```

Comment:
We don't want to prevent default if `addOnPaste` is set to `false`

Reply:
yes i changed it to : on Line No 328 
```
     if (addOnPaste) {
	e.preventDefault();
     }
```

Reply:
That will always be true, as that is the store, not the value of the store. We're also already checking for the value a few lines below.

Reply:
Can i do this instead 
```
                     if (!get(addOnPaste)) return;
                     else e.preventDefault();
```